build/packaging/linux/bin/kend

```diff
@@ -220,6 +309,14 @@ start() {
     return $RETVAL
 }
 
+start_docker() {
```

Comment:
Multiple KES docker can run at the same time. Is this intended? @aidan-kwon @KimKyungup